Source/WebCore/loader/PingLoader.cpp

```diff
@@ -186,7 +191,7 @@ void PingLoader::sendViolationReport(Frame& frame, const URL& reportURL, Ref<For
 
     HTTPHeaderMap originalRequestHeader = request.httpHeaderFields();
 
-    if (reportType != ViolationReportType::StandardReportingAPIViolation && reportType != ViolationReportType::Test)
+    if (reportType != ViolationReportType::StandardReportingAPIViolation && reportType != ViolationReportType::Test && reportType != ViolationReportType::CrossOriginOpenerPolicy)
```

Comment:
Maybe we should check check for equality with the CSP ViolationReportType, unless we have other types that need this behavior.